ModernFlyouts/MultilingualResources/ModernFlyouts.nl.xlf

```diff
@@ -8,429 +8,429 @@
+          <target state="final" state-qualifier="mt-suggestion">Vergrendelingstoetsen Flyout aanzetten</target>
         </trans-unit>
         <trans-unit id="Settings.FlyoutTheme" translate="yes" xml:space="preserve">
           <source>Flyout theme</source>
-          <target state="needs-review-translation" state-qualifier="mt-suggestion">Thema Flyout</target>
+          <target state="final" state-qualifier="mt-suggestion">Flyout thema</target>
         </trans-unit>
         <trans-unit id="Settings.FlyoutTimeout" translate="yes" xml:space="preserve">
           <source>Flyout timeout (ms)</source>
-          <target state="needs-review-translation" state-qualifier="mt-suggestion">Time-out voor flyout (ms)</target>
+          <target state="final" state-qualifier="mt-suggestion">Flyout time-out (ms)</target>
         </trans-unit>
         <trans-unit id="Settings.Left" translate="yes" xml:space="preserve">
           <source>Left</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">Links</target>
+          <target state="final" state-qualifier="tm-suggestion">Links</target>
         </trans-unit>
         <trans-unit id="Settings.Modules.Audio.Media" translate="yes" xml:space="preserve">
           <source>Media</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">Medium</target>
+          <target state="final" state-qualifier="tm-suggestion">Media</target>
         </trans-unit>
         <trans-unit id="Settings.Modules.Audio.Volume" translate="yes" xml:space="preserve">
           <source>Volume</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">Jaargang</target>
+          <target state="final" state-qualifier="tm-suggestion">Volume</target>
         </trans-unit>
         <trans-unit id="Settings.Right" translate="yes" xml:space="preserve">
           <source>Right</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">Rechts</target>
+          <target state="final" state-qualifier="tm-suggestion">Rechts</target>
         </trans-unit>
         <trans-unit id="Settings.TopBarVisibility" translate="yes" xml:space="preserve">
           <source>TopBar visibility</source>
-          <target state="needs-review-translation" state-qualifier="mt-suggestion">Zichtbaarheid van topbar</target>
+          <target state="final" state-qualifier="mt-suggestion">Zichtbaarheid van de bovenste balk</target>
         </trans-unit>
         <trans-unit id="Settings.TrayIconEnabled" translate="yes" xml:space="preserve">
           <source>Show icon on system tray area</source>
-          <target state="needs-review-translation" state-qualifier="mt-suggestion">Pictogram weergeven op systeemladegebied</target>
+          <target state="final" state-qualifier="mt-suggestion">Pictogram weergeven op systeemvak</target>
```

Comment:
I have never heard this word used before, always system tray, but I think some people won't understand system tray.